src/screens/Products/AddProduct.js

```diff
@@ -0,0 +1,310 @@
+import { AddImage, AsyncButton, UserInput, Wallpaper } from '@components';
+import React, { Component } from 'react';
+import { StyleSheet, ScrollView, View, Text, TextInput, Picker } from 'react-native';
+import ImagePicker from 'react-native-image-crop-picker';
+
+export default class AddProduct extends Component {
+  static navigationOptions = {
+    title: 'Add a Handmade Product'
+  }
+
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      productMainCategory: 'Handmade Apparel',
+      productSubCategory: 'Activewear',
+
+      productMainPicture: "",
+      productTitle: "",
+      productDescription: "",
+      productGender: 'Women',
```

Comment:
This is a dropdown menu and the default value is Women if nothing is changed or selected.

Reply:
Possibly look into a default label for the picker which can be customized